activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -248,7 +249,19 @@ def [](arg)
       # A convenience method for returning a collection of TimeZone objects
       # for time zones in the USA.
       def us_zones
-        @us_zones ||= all.find_all { |z| z.name =~ /US|Arizona|Indiana|Hawaii|Alaska/ }
+        country_zones('US')
```

Comment:
Sure we can. Anyway it will be converted to string and upcased. Will push fix in a minute.